### ISSUE TITLE
WCAG 1.4.3: Ensure the contrast between foreground and background colors meets WCAG 2 AA minimum contrast ratio thresholds (.hover\:bg-destructive\/90)

### ISSUE THREAD

**Issue by azap026** [azap026/smetalabv3#86]
#### Issue
`Ensure the contrast between foreground and background colors meets WCAG 2 AA minimum contrast ratio thresholds` ([color-contrast](https://accessibilityinsights.io/info-examples/web/color-contrast))
#### Target application
[Smetalab - Smart Engineering & Management](http://localhost:3000/app/guide/works)
#### Element path
.hover\:bg-destructive\/90
#### Snippet
`<button data-slot="tooltip-trigger" data-variant="destructive" data-size="default" class="inline-flex items-center justify-center gap-2 whitespace-nowrap rounded-md font-medium transition-all disabled:pointer-events-none disabled:opacity-50 [&amp;_svg]:...`
#### How to fix
    Fix any of the following:
      Element has insufficient color contrast of 3.76 (foreground color: #ffffff, background color: #ef4444, font size: 10.5pt (14px), font weight: normal). Expected contrast ratio of 4.5:1
#### Environment
Chrome version 143.0.0.0
This accessibility issue was found using Accessibility Insights for Web 2.46.0 (axe-core 4.10.2), a tool that helps find and fix accessibility issues. Get more information & download this tool at [http://aka.ms/AccessibilityInsights](http://aka.ms/AccessibilityInsights).
